sympy/tensor/tensor.py

```diff
@@ -299,7 +294,7 @@ def indices_canon_args(self):
         """
         Returns ``(g, dummies, msym, v)``, the entries of ``canonicalize``
 
-        see ``canonicalize`` in ``tensor_can.py``
+        see ``canonicalize`` in ``tensor_can.py`` in combinatorics module
```

Comment:
I think we have some way to cross-link the documentation. I don't remember exactly how.